extensions/sql-migration/src/constants/iconPathHelper.ts

```diff
@@ -128,5 +129,9 @@ export class IconPathHelper {
 			light: context.asAbsolutePath('images/sendFeedback.svg'),
 			dark: context.asAbsolutePath('images/sendFeedback.svg')
 		};
+		IconPathHelper.emptyTable = {
+			light: context.asAbsolutePath('images/emptyTable.svg'),
+			dark: context.asAbsolutePath('images/emptyTable.svg')
```

Comment:
Can you take a screenshot of how this icon shows up on both light and dark themes in ADS?